packages/uniforms-mantine/src/NestField.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
```

Comment:
in the multiple files you imported `React`, but it isn't used

Reply:
It was checked on code with a different eslint config. `React` import is needed here.